js/app.js

```diff
@@ -113,6 +113,14 @@ function App() {
 			moduleFolder = `${__dirname}/../modules/default/${module}`;
 		}
 
+		const moduleFile = `${moduleFolder}/${module}.js`;
+
+		try {
+			fs.accessSync(moduleFile, fs.R_OK);
+		} catch (e) {
+			Log.error(`No ${moduleFile} found for module: ${moduleName}.`);
```

Comment:
Maybe warn level would be sufficient here?

Reply:
I don't care at all, no problem to change ...

Reply:
done.